### PR TITLE
phactory: add "WrapBytes" signature types

### PR DESCRIPTION
**Pull request by h4x3rotab**
Companion proto PR: https://github.com/Phala-Network/prpc-protos/pull/9

This PR introduces a new kind of SignatureType: add `<Bytes>` and `</Bytes>` to wrap the message to sign, called "Wrap Bytes" in Polkadot.js Extension. We have to support it because all the messages signed by Polkadot.js Extension are in this format.

See more at: https://github.com/polkadot-js/extension/blob/e4ce268b1cad5e39e75a2195e3aa6d0344de7745/packages/extension-dapp/src/wrapBytes.ts

### CODE REVIEW

File: crates/phactory/src/prpc_service.rs [hunk 1]
```diff
@@ -478,47 +479,6 @@ impl<Platform: pal::Platform> Phactory<Platform> {
         Ok(fit_size(messages, output_buf_len))
     }
 
-    fn send_coordinate_info(&mut self, request: pb::SendCoordinateInfoRequest) -> RpcResult<()> {
-        // local ecdh_key
```
Comment (kvinwang) [inline crates/phactory/src/prpc_service.rs:482]: Isn't this change belong to #498 ?

Comment (h4x3rotab) [reply crates/phactory/src/prpc_service.rs:482]: I cannot compile if I don't remove it. Do you suggest to checkout an older version of the proto repo?

Comment (h4x3rotab) [reply crates/phactory/src/prpc_service.rs]: Rolled back to an earlier version

**Review by kvinwang — APPROVED** [review]
LGTM

**Pull request merged**